rust/metaplex/program/src/instruction.rs

```diff
@@ -182,20 +184,21 @@ pub enum MetaplexInstruction {
     ///   5. `[]` Vault account
     ///   6. `[]` Fraction mint of the vault
     ///   7. `[]` Auction
-    ///   8. `[]` Your BidderMetadata account
-    ///   9. `[signer optional/writable]` Your Bidder account - Only needs to be signer if payer does not own
-    ///   10. `[signer]` Payer
-    ///   11. `[]` Token program
-    ///   12. `[]` Token Vault program
-    ///   13. `[]` Token metadata program
-    ///   14. `[]` Store
-    ///   15. `[]` System
-    ///   16. `[]` Rent sysvar
-    ///   17. `[signer]` Transfer authority to move the payment in the auction's token_mint coin from the bidder account for the participation_fixed_price
+    ///   8. `[]` Auction extended
```

Comment:
Yah again, can't do this - you'll break older UIs. Got to put it at the end, or make a new end point.